core/src/main/resources/com/squarespace/template/plugins/platform/variants-select.html

```diff
@@ -12,7 +12,8 @@
         <option value="">{selectText|message variantName:name}</option>
         {.repeated section values}<option value="{@|htmltag}">{@|htmltag}</option>{.end}
       </select>
-    </div>
+    </div>{.repeated section values}
+    <div class="variant-box" value="{@|htmltag}">{@|htmltag}</div>{.end}
```

Comment:
Will you not need a <div> surrounding the array of divs for any styling purposes?

Reply:
yeah, that's a good idea, I was thinking I could use the existing `product-variants` container but having a separate container keeps the styles better encapsulated.